cmd/manager/cmd/config.go

```diff
@@ -0,0 +1,114 @@
+package cmd
+
+import (
+	"errors"
+	"io/ioutil"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/spf13/cobra"
+	"github.com/ugcompsoc/apid/internal/config"
+)
+
+// configCmd represents the config command
+var configCmd *cobra.Command
+
+func NewConfigCmd() *cobra.Command {
+	configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Displays a config given a directory",
+		Long: `Prints out the config in the default directory if no directory is specified.
+If a directory is specified it will look for a compatible file in there and
+will print it out instead.`,
+		Run: VerifyConfig,
+	}
+
+	configCmd.AddCommand(NewCreateConfigCmd())
+	configCmd.PersistentFlags().StringP("directory", "d", ".", "Directory for config")
+	configCmd.PersistentFlags().StringP("filename", "f", "apid.yml", "filename for config")
+	configCmd.PersistentFlags().BoolP("print", "p", false, "Print config")
+	configCmd.PersistentFlags().BoolP("secrets", "s", false, "Print secrets")
+
+	return configCmd
+}
+
+func verifyFileAndExtract(file []byte) (*config.Config, []string, error) {
+	fileStr := string(file)
+	if len(fileStr) == 0 {
+		return nil, nil, errors.New("The file apid.yml is completely empty. What do you want me to do with this?")
+	}
+	c := config.Config{}
+	err := yaml.Unmarshal(file, &c)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	issues, err := c.Verify()
+	return &c, issues, err
+}
+
+func VerifyConfig(cmd *cobra.Command, args []string) {
+	debug, _ := cmd.Flags().GetBool("debug")
+	directory, _ := cmd.Flags().GetString("directory")
+	filename, _ := cmd.Flags().GetString("filename")
```

Comment:
TODO for me: Add a check for the filename here like there is https://github.com/ugcompsoc/apid/pull/49/files#diff-30d7eff799d9c86d06d11ec0ce760421c01e24869471ed731a5370c035b29ee6R99